### ISSUE TITLE
Dashboard Gauge - Issue if using Math Measurement

### ISSUE THREAD

**Issue by mrplow** [kizniche/Mycodo#461]
## Mycodo Issue Report:
  - Specific Mycodo Version: 6.0.8
 
#### Problem Description
Please list:

  - I have a BME280. I created a new Math equation measurement to convert it's pressure readings from Pa to kPa (x/100). The kPa measurement does work on the Live page and the Dashboard if I add it to a graph. But after I add it to a Gauge all of my dashboard measurements are showing as empty.
 
### Errors
  - Inspecting the element is Chrome revealed: dashboard:2896 Uncaught SyntaxError: Invalid or unexpected token.
 - Looks like there is an extra carriage return in the name field and the data is _null_:
```
    series: [{
        name: '
            pressure',
        data: [null],
        dataLabels: {
          style: {"fontSize": "14px"},
          format: '{point.y:,.1f}'
        },
```

### Steps to Reproduce the issue:
How can this issue be reproduced?
 1. Create a math input
 2. Add that input to a Gauge
 
### Additional Notes
This project is awesome

**Comment by kizniche** [comment]
Just released v6.0.9 with the fix for this. Thanks for the bug report.

**Comment by mrplow** [comment]
Working great, thanks
